scripts/mem_usage.py

```diff
@@ -88,6 +88,15 @@ def main():
     args = get_args()
     env = os.environ.copy()
     env['LC_ALL'] = 'C'
+    readelf = subprocess.Popen(str.split(readelf_cmd()) + ['-s',
+                                                           args.tee_elf],
+                               stdout=subprocess.PIPE, env=env,
+                               universal_newlines=True)
+    for line in iter(readelf.stdout.readline, ''):
+        words = line.split()
+        if len(words) == 8 and words[7] == '_end_of_ram':
+            end_of_ram = int(words[1], 16)
+            break
     readelf = subprocess.Popen(str.split(readelf_cmd()) + ['-S', '-W',
```

Comment:
Shouldn't `readelf` be close before reassigned or is that done automatically?

Reply:
I suppose it's not closed and both processes are killed when the script exits. I can add `readelf.terminate()` if you prefer.

Reply:
Yes please, I think that's more clear.